src/popup/router/pages/Networks.vue

```diff
@@ -94,6 +107,7 @@ const networkProps = {
   url: null,
   middlewareUrl: null,
   compilerUrl: null,
+  backendUrl: 'https://raendom-backend.z52da5wt.xyz',
```

Comment:
why is it hardcoded here?

Reply:
I could export it in a constant and import it here. This is our default backend url, the user has the option to change it. Do you think I should leave the field blank?